snapcraft/_options.py

```diff
@@ -137,13 +137,14 @@ def snap_dir(self):
         return os.path.join(self.__project_dir, 'prime')
 
     def __init__(self, use_geoip=False, parallel_builds=True,
-                 target_deb_arch=None):
+                 target_deb_arch=None, project_type="app"):
```

Comment:
I'll actually remove this since we no longer retain ownership conditionally based on the snap type.